views/authentication.blade.php

```diff
@@ -1,63 +1,70 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
-        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
-<html>
+<!DOCTYPE html>
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
 <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-    <meta http-equiv="Content-Language" content="fr" />
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>{{ config('app.name', 'Laravel') }}</title>
     <script src="{!! secure_asset('vendor/u2f/u2f.js') !!}"></script>
     <script src="{!! secure_asset('vendor/u2f/app.js') !!}"></script>
-    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" rel="stylesheet">
+
+    <!-- Styles -->
+    <link href="{{ asset('css/app.css') }}" rel="stylesheet">
```

Comment:
This change assum that the project keep the default bootstrap.

Reply:
Yes, it's the most "touchy" part of this PR, maybe I should add a way to publish views. I assume by default we use Bootstrap 4, provided by Laravel.

Reply:
There is a configuration to change the view.
https://github.com/lahaxearnaud/laravel-u2f/blob/master/config/u2f.php#L33

But it would be nice to publish views with something like this in th procider:

```php
$this->publishes([
        __DIR__.'/../../views' => resource_path('views/vendor/laravel-u2f'),
    ]);
 ```